libc/LibC.h

```diff
@@ -67,7 +67,7 @@ Core context handling macros, only intended to be called from other macros
 ***********************************************************************************************************************************/
 #define MEM_CONTEXT_XS_CORE_BEGIN(memContext)                                                                                      \
     /* Switch to the new memory context */                                                                                         \
-    MemContext *MEM_CONTEXT_memContextPrior = memContextSwitch(memContext);                                                        \
+    memContextPush(memContext);                                                                                                    \
```

Comment:
If MEM_CONTEXT_XS_CORE_BEGIN does memContextPush then why would MEM_CONTEXT_XS_CORE_END no do a memContextPop?

Reply:
Good catch. It should, but this macro is dead code so it doesn't really matter what the contents are.  It must have become dead during the cleanup after the Perl migration was complete.

I have removed the code from the master branch in this commit: https://github.com/dwsteele/pgbackrest-dev/commit/7866861b9f61c12b63e0b1cb83ce0b78e0611780.  Since MEM_CONTEXT_XS_CORE_BEGIN() is now only used in one place I merged it into MEM_CONTEXT_XS_TEMP_BEGIN().  If that looks OK I'll push to master and update the trymem branch accordingly.